eks-anywhere-common/Addons/Partner/dynatrace/dynatrace-variables.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dynatrace-variables
+  namespace: flux-system
```

Comment:
I believe they need to be in flux-system in order for flux to have access to the values to do the substitution? but I don't know this for a fact.

Reply:
this question goes hand in hand with the question of the namespace for Kustomization [objects](https://github.com/aws-samples/eks-anywhere-addons/pull/31/files#diff-d4f24a6bdfb1afeea9b027e68614701cafc59085c491fc0e08f85d39e3d0a43dR32). Since kustomization is in flux-system the config-map should also be there.